packages/core/test-support/helpers/contains-helpers.js

```diff
@@ -27,3 +27,62 @@ export function buttonContains(text) {
 export function linkContains(text) {
   return findByContains('a', text);
 }
+
+/**
+ * This function allows us to keep using the jquery notation for :contains without using jquery
+ *
+ * Example:
+ * Jquery: $(.class-thing:contains(blah) .metric .test:contains(other))
+ *
+ * findByContains:
+ * [...findByContains('.class-thing', 'blah').querySelectorAll('.metric .test')].find(el => el.innerText.trim().includes('other'))
+ *
+ * Non-Jquery Equivalent: [
+ *    ...[
+ *      ...document.querySelectorAll(
+ *        (.class-thing)
+ *      )].find(el => el.innerText.trim().includes('blah'))
+ *    .querySelectorAll('.metric .test')
+ *  ]
+ *  .find(el => el.innerText.trim().includes('other'))
+ *
+ * findContains: findContains(.class-thing:contains(blah) .metric .test:contains(other))
+ *
+ * @param {String} selector - jquery or native style selector
+ * @param {HTMLElement} [baseElement] - optional parameter to use as base element to select off of
+ * @returns {HTMLElement} - Element based on text it (or its parent) contains
+ */
+export function findContains(selector, baseElement) {
+  let nativeSelector,
+    parentElement = baseElement || document;
+
+  if (selector.includes(':contains(')) {
+    let splits = selector.split(':contains(');
+    let firstSelector = splits.shift(); //Selector without the :contains operator
+    let restOfSelector = splits.join(':contains('); //We only want to split on the first instance of :contains(, so join the rest back together if there were more
+    let endOfContentIndex = findEndOfContentIndex(restOfSelector);
+    let content = restOfSelector.substring(0, endOfContentIndex);
+    let nextSelector = restOfSelector.substring(endOfContentIndex + 1).trim();
+    let filteredElement = [...parentElement.querySelectorAll(firstSelector)].find(el =>
+      el.innerText.trim().includes(content)
+    );
```

Comment:
Why not use findByContains here?

Reply:
We're going to get rid of findByContains